packages/openstef-core/src/openstef_core/feature_engineering/forecasting_transforms/scaler.py

```diff
@@ -0,0 +1,121 @@
+# SPDX-FileCopyrightText: 2025 Contributors to the OpenSTEF project <short.term.energy.forecasts@alliander.com>
+#
+# SPDX-License-Identifier: MPL-2.0
+
+"""Transform for scaling features in time series data.
+
+This module provides data scaling functionality using various scaling methods
+from scikit-learn to normalize and standardize features in time series datasets
+for improved machine learning model performance.
+"""
+
+from enum import StrEnum
+
+import pandas as pd
+
+from openstef_core.datasets import TimeSeriesDataset
+from openstef_core.datasets.transforms import TimeSeriesTransform
+
+try:
+    from sklearn.preprocessing import MaxAbsScaler, MinMaxScaler, RobustScaler, StandardScaler
+except ImportError as e:
+    raise ImportError(
+        "scikit-learn is required for the Scaler transform. Please install it via "
+        "`uv sync --group ml --package openstef-core` or `uv sync --all-groups --package openstef-core`."
+    ) from e
+
+
+class ScalingMethod(StrEnum):
+    """Scaling methods from sklearn.preprocessing."""
+
+    MinMax = "min-max"
+    MaxAbs = "max-abs"
+    Standard = "standard"
+    Robust = "robust"
+
+
+class Scaler(TimeSeriesTransform):
```

Comment:
Suffixing the transforms would probably more clear what this component is. For example `ScalerTransform`

Reply:
Fixed